src/skai/generate_examples.py

```diff
@@ -95,7 +95,7 @@ class ExamplesGenerationConfig:
   labels_file: Optional[str] = None
   label_property: Optional[str] = None
   labels_to_classes: Optional[List[str]] = None
-  num_keep_labeled_examples: int = 1000
+  num_keep_labeled_examples: int = None
```

Comment:
This line is also causing a test failure:

Traceback (most recent call last):
  File "/tmpfs/src/github/skai/src/skai/generate_examples_test.py", line 424, in testConfigLoadedCorrectlyFromJsonFile
    self.assertEqual(config.num_keep_labeled_examples, 1000)
AssertionError: None != 1000